concourse/pipelines/guest-package-build.jsonnet

```diff
@@ -474,6 +511,13 @@ local build_guest_agent = buildpackagejob {
             machine_type: 't2a-standard-2',
             worker_image: 'projects/compute-image-tools/global/images/family/debian-11-worker-arm64',
           },
+          buildpackageimagetaskcos {
+            image_name: 'cos-113',
```

Comment:
Should be we following COS-stable instead, or are we expecting to add more families with new releases of COS versions?

Reply:
I think for testing purposes it might make sense to start off with cos-113-lts, WDYT? If we can get this to work in the pipeline then ideally we would like to test on all active COS versions (ie 101, 105, 109, 117).

Reply:
sgtm